src/pages/Search/SearchPage.tsx

```diff
@@ -17,10 +18,11 @@ type SearchPageProps = StackScreenProps<AuthScreensParamList, typeof SCREENS.SEA
 function SearchPage({route}: SearchPageProps) {
     const {shouldUseNarrowLayout} = useResponsiveLayout();
     const styles = useThemeStyles();
+    const {policyIDs, q} = route.params;
 
-    const {policyIDs} = route.params;
+    const isCustomQueryMode = SearchUtils.isCustomQuery(route.params);
```

Comment:
Let's rename all of these instances
```suggestion
    const isCustomQuery = SearchUtils.isCustomQuery(route.params);
```